### ISSUE TITLE
Parsing error with `<script>` in a string literal

### ISSUE THREAD

**Issue by aradalvand** [sveltejs/eslint-plugin-svelte#578]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.48.0

### What version of `eslint-plugin-svelte` are you using?

2.33.0

### What did you do?

I'm using Partytown with SvelteKit based on the [official guide](https://partytown.builder.io/sveltekit#1-add-the-partytown-script-to-srcrouteslayoutsvelte), here's the code I have in my `+layout.svelte`:

```svelte
<script>
  import { partytownSnippet } from '@builder.io/partytown/integration'
</script>

<svelte:head>
  {@html `<script>${partytownSnippet()}</script>`}
</svelte:head>
```

`eslint-plugin-svelte` seems to get confused at the `<script>` part inside the string literal; I get the following error:
> Parsing error: Unexpected keyword or identifier. eslint

### What did you expect to happen?

The `<script>` inside the string should be recognized as a string.

### What actually happened?

It seems to be confused by the `<script>` part, it may worth noting that if you changed `<script>` to any other tag like `<style>` here the error would go away, so the issue must be specific to `<script>`.

### Link to **GitHub Repo** with Minimal Reproducible Example

Repro: https://github.com/aradalvand/eslint-plugin-svelte-bug

Go to `src/routes/+layout.svelte` — you'll see the error:

![image](https://github.com/sveltejs/eslint-plugin-svelte/assets/26527405/71cad7c1-ebf0-4be5-a770-fcc4ac18e8a6)

**Comment by baseballyama** [comment]
Duplicate of https://github.com/sveltejs/eslint-plugin-svelte/issues/562

**Comment by aradalvand** [comment]
@baseballyama I don't think so, this is working properly Svelte-wise.
